x-pack/plugins/observability_solution/metrics_data_access/server/lib/metrics/index.ts

```diff
@@ -39,6 +39,14 @@ export const query = async (
     },
   };
   const hasGroupBy = Array.isArray(options.groupBy) && options.groupBy.length > 0;
+  const groupInstanceFilter =
+    options.groupInstance?.reduce<Array<Record<string, any>>>((acc, group, index) => {
```

Comment:
Either refine to a more appropriate type or use `unknown` if we don't care about the type

```suggestion
    options.groupInstance?.reduce<Array<Record<string, unknown>>>((acc, group, index) => {
```

Reply:
Fixed in [5ff89e3](https://github.com/elastic/kibana/pull/181689/commits/5ff89e38d702807e6f1171b0e546047a87d15b40)